code/modules/clothing/glasses/glasses.dm

```diff
@@ -149,6 +149,19 @@ var/list/science_goggles_wearers = list()
 	icon_state = "eyepatch"
 	item_state = "eyepatch"
 	min_harm_label = 0
+	var/flipped = FALSE
+
+/obj/item/clothing/glasses/eyepatch/attack_self()
+	if(!flipped)
```

Comment:
This entire statement can be simplified.

```
.../attack_self(mob/user)
 flipped = !flipped
 icon_state = "eyepatch[flipped]"
 item_state = "eyepatch[flipped]"
 to_chat(user, "You flip \the [src] to your [flipped ? "left" : "right"] eye.")
```

Note that I added the arg mob/user and used it for to_chat here. This will cover you for both left and right with no need for paste or an if/else (since we cover that with the ternary). Also note you will need to rename eyepatches to "eyepatch0" and "eyepatch1" (right and left)

Reply:
Neat, thank you.